pkg/asset/installconfig/platform.go

```diff
@@ -119,7 +120,7 @@ func (a *Platform) queryUserForPlatform() (string, error) {
 
 	platform, err := prompt.Generate(nil)
 	if err != nil {
-		return "", err
+		return "", errors.Wrapf(err, "failed to Generate %s", prompt.Name())
```

Comment:
If you feel the need for more specific errors like this, it's DRYer to address them in the `Generate` function itself (which also has access to the prompt name).

Reply:
The context seems correct that Generate failed on *the* prompt.

Reply:
> The context seems correct that Generate failed on *the* prompt.

Right, but you can add that context within `UserProvided.Generate` itself.  For example, with eff2fdce1e and a ctrl-C:

```console
$ openshift-install install-config
? Platform  [Use arrows to move, type to filter]
> aws
  libvirt
  openstack
FATAL Failed to generate Install Config: failed to fetch dependency for Install Config: failed to generate asset Platform: failed to acquire user-provided input Platform: failed to Ask: interrupt 
```

Reply:
Sure